invictum-test-core/src/main/java/com/github/invictum/locator/factory/providers/JqueryLocatorProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.github.invictum.locator.factory.providers;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.By;
+
+public class JqueryLocatorProvider implements LocatorProvider {
+
+    @Override
+    public boolean isAcceptableTo(String locator) {
+        return locator.matches("^jquery(.+?|)=.+$");
```

Comment:
This patter allow locators like:
jquerySOME OTHER TEXT = path
Maybe it will be better to replace it with something like `"^jquery( |)=.+$"`